rest-api/src/main/resources/definitions/adherence_report_search.yml

```diff
@@ -0,0 +1,37 @@
+type: object
+properties:
+    testFilter:
+        description: Return test, production, or both kinds of records. If the caller is a developer without access to production records, this field will be ignored and only test records will be returned.
+        $ref: ./enums/test_filter.yml
+    labelFilters:
+        description: Return records where the labels of the entries includes one or more of the labels in this array.
+        type: array
+        items:
+            type: string
+    adherenceMin:
+        description: Only return records with a weekly adherence at or above this value.
+        type: integer
+        format: int32        
+    adherenceMax:
+        description: Only return records with a weekly adherence at or below this value.
+        type: integer
+        format: int32        
+    progressionFilters:
+        description: Filter records by whether the participant has finished, not yet begun, and/or is currently participating in the study.
+        type: array
+        items:
+            $ref: ./enums/participant_study_progress.yml
```

Comment:
In the server, this is progressionFilter (singular). It's not a bad idea to switch it to an array though. If someone wants to search for both "unstarted" and "in_progress" at the same time, for instance.

Reply:
This has been changed to be an array of values, plural (progressionFilters) on the server, based on the CR when it became apparent it would be useful.